src/components/tasks/SuggestionBox/Suggestions.tsx

```diff
@@ -0,0 +1,75 @@
+import { forwardRef } from 'react';
+import classNames from '@/components/tasks/card/card.module.scss';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import { userDataType } from '@/interfaces/user.type';
+import { Loader } from '../card/Loader';
+import SuggestionBox from '../SuggestionBox/SuggestionBox';
+import { SuggestionsProps } from '@/interfaces/suggestionBox.type';
+import { useGetAllUsersByUsernameQuery } from '@/app/services/usersApi';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+
+const Suggestions = forwardRef<HTMLInputElement, SuggestionsProps>(
+    (
+        {
+            assigneeName,
+            searchTerm,
+            showSuggestion,
+            handleAssignment,
+            handleChange,
+            handleClick,
+        },
+        ref
+    ) => {
+        const { data, isLoading } = useGetAllUsersByUsernameQuery(
+            {
+                searchString: searchTerm,
+            },
+            { skip: searchTerm ? false : true }
+        );
+
+        const usersData = data?.users;
```

Comment:
```suggestion
        const { users } = data;
```

Reply:
I have to do optional chaining, because otherwise it throws error that the userResponse doesnt exist